### PR TITLE
Bump nginx ingress version

### PR DESCRIPTION
**Pull request by stone-z**
Bump nginx ingress version to newest upstream for 1.15.4 release

### CODE REVIEW

**Comment by stone-z** [comment]
Note: 0.26.0 had the following breaking changes (no additional were reported for 0.26.1):
- The variable $the_real_ip variable was removed from template and default log_format.
- The default value of configmap setting proxy-add-original-uri-header is now "false".
- When the setting proxy-add-original-uri-header is "true", the ingress controller adds a new header X-Original-Uri with the value of NGINX variable $request_uri.
- In most of the cases this is not an issue but with request with long URLs it could lead to unexpected errors in the application defined in the Ingress serviceName, like issue 4593 - 431 Request Header Fields Too Large

**Comment by pipo02mix** [comment]
Have you tested with our current active version?

About the breaking changes, I would add a note in release notes mentioning there are some minor braking changes and link to release page of the upstream nginx version

**Review by rossf7 — APPROVED** [review]
LGTM when tested and @pipo02mix is happy.

**Review by pipo02mix — APPROVED** [review]
if it is tested and worked fine, I am good

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project's packages adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.11.0]
+
+### Breaking Changes
```
Comment (kopiczko) [inline CHANGELOG.md]:
```suggestion
### Changed
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,17 @@ All notable changes to this project will be documented in this file.
...
+
+- Upgrade nginx controller to use the latest version [0.26.1](https://github.com/kubernetes/ingress-nginx/releases/tag/nginx-0.26.1).  
+**This version includes the following breaking changes**:
+   - The variable $the_real_ip variable was removed from template and default log_format.
+   - The default value of configmap setting proxy-add-original-uri-header is now "false".
+   - When the setting proxy-add-original-uri-header is "true", the ingress controller adds a new header X-Original-Uri with the value of NGINX variable $request_uri.
+   - In most of the cases this is not an issue but with request with long URLs it could lead to unexpected errors in the application defined in the Ingress serviceName, like issue 4593 - 431 Request Header Fields Too Large
```
Comment (kopiczko) [inline CHANGELOG.md]: Can it just point to breaking changes somewhere else? Like in upstream changelog?

Comment (rossf7) [reply CHANGELOG.md]: There is a lot of detail here but its also a tradeoff. We've had problems multiple times with breaking changes where customers said it wasn't clear enough in the release notes. 

@puja108 @pipo02mix WDYT?

Comment (pipo02mix) [reply CHANGELOG.md]: yeah I have my doubts, but if we are repeating the same info as it is in the upstrem nginx release info then I will just link it. But in the GS release I will highlight it so customer check the page before upgrading, same for Release notes (slack)

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project's packages adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.11.0]
```
Comment (TheoBrigitte) [inline CHANGELOG.md]: When this version introduce a breaking change, we should bump major version to `1.0.0`.

**Pull request merged**